src/main/static/src/components/kutsuminen/KutsuConfirmation.js

```diff
@@ -63,9 +57,14 @@ export default class KutsuConfirmation extends React.Component<Props, State> {
                     <div className="row">
                         {this.state.sent
                             ? <Button action={this.onClose.bind(this)}>{L['VIRKAILIJAN_LISAYS_LAHETETTY']}</Button>
-                            : <Button action={this._sendInvitation.bind(this)}>{L['VIRKAILIJAN_LISAYS_TALLENNA']}</Button>
+                            : <Button action={this._sendInvitation.bind(this)} loading={this.state.loading} disabled={this.state.loading}>{L['VIRKAILIJAN_LISAYS_TALLENNA']}</Button>
```

Comment:
disabled={this.state.loading} voisi laittaa suoraan Button:in sisään disabled-kenttään jos ei jossain ole tarvetta painaa lataa-tilassa olevaa nappia.